libs/core/src/lib/action-sheet/action-sheet.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { EventEmitter, InjectionToken } from '@angular/core';
+
+export const ACTION_SHEET_COMPONENT = new InjectionToken<string[]>('ActionSheetComponent');
+
+/**
+ * TODO
```

Comment:
Add some description there, just if you decide to use approach I presented to you